src/library/Shared/Implementations/ExtendedSplashScreen.Android.cs

```diff
@@ -60,19 +61,29 @@ private static void OnExitAnimation(object sender, AndroidX.Core.SplashScreen.Sp
 		{
 			Trace("Started OnExitAnimation.");
 
-			_splashBitmap = GetBitmapFromView(e.SplashScreenViewProvider.View);
+			var view = e.SplashScreenViewProvider.View;
+
+			// It seems like the view can sometimes be invalid according to https://github.com/nventive/ExtendedSplashScreen/issues/76.
+			_isNativeViewValid = view.Width > 0 && view.Height > 0;
+
+			if (_isNativeViewValid.Value)
+			{
+				_splashBitmap = GetBitmapFromView(view);
+			}
+			else
+			{
+				Trace("Native view is invalid because at least one of its dimension is 0.");
```

Comment:
could we also log the view dimensions to have a bit more information?

Reply:
I don't think it's relevant. The problem is that there's a 0 in there and that shouldn't be the case.